openmetadata-ui/src/main/resources/ui/cypress/e2e/AddNewService/redshiftWithDBT.spec.js

```diff
@@ -0,0 +1,169 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {
+    deleteCreatedService,
+    editOwnerforCreatedService,
+    goToAddNewServicePage,
+    interceptURL,
+    login, testServiceCreationAndIngestion,
+    updateDescriptionForIngestedTables,
+    uuid,
+    verifyResponseStatusCode,
+    visitEntityDetailsPage
+} from '../../common/common';
+import {
+    DBT, HTTP_CONFIG_SOURCE, LOGIN,
+    SERVICE_TYPE
+} from '../../constants/constants';
+
+const serviceType = 'Redshift';
+const serviceName = `${serviceType}-ct-test-${uuid()}`;
+const tableName = 'boolean_test';
+const DBTTable = 'customers';
+const description = `This is ${serviceName} description`;
```

Comment:
Can we move them to const?
If needed split the constant file to something like: `services.constant.ts` & `common.constant.ts`